extensions/gadgets/base.py

```diff
@@ -84,7 +84,8 @@ def html_body(self):
         once the necessary attributes are supplied.
         """
         html_templates = utils.get_file_contents(os.path.join(
-            feconf.GADGETS_DIR, self.type, '%s.html' % self.type))
+            utils.get_asset_dir_prefix()[1:], feconf.GADGETS_DIR,
```

Comment:
Previously the gadgets were loaded were loaded from same folder (oppia/extensions/gadegets/…) in dev mode and in prod mode too. Now, because the HTMLs are modified in prod mode (the URLs are replaced with URLs with hashes), we need to load them from build folder. 

The gadgets should probably be served statically, also the #2575 probably should have been solved before this.

(all stated for gadgets apply to RTCs and interactions too)